image/templates/helm/stackrox-secured-cluster/README.md.htpl

```diff
@@ -250,7 +250,7 @@ chart and their default values:
 |`image.main.registry`| Address of the registry you are using for main image.|`[< required "" .MainRegistry >]` |
 |`image.collector.registry`| Address of the registry you are using for collector image.|`[< required "" .CollectorRegistry >]` |
 |`sensor.endpoint`| Address of the Sensor endpoint including port number. No trailing slash.|`sensor.stackrox.svc:443` |
-|`collector.forceCollectionMethod`|Enabling this setting overrides the default collection method with the content of `collector.collectionMethod` |`false` |
+|`collector.forceCollectionMethod`|Deprecated. This value has no effect. |`false` |
```

Comment:
Will this line then be removed in a later release?

Reply:
Yeah that was my intention. I wanted to keep it documented for this release for those customers that have been using it in 4.4, rather than just removing it outright